linode/manager
# chore: [M3-6777] - Rename Cypress functions and variables related to secrets

## Description 📝
This aims to clean up some of our CodeQL alerts which are caused by using JavaScript's math functions to generate random values. Because some of our tests involve mocking secrets and keys, CodeQL thinks we're generating actual secrets and keys using insecure functions and flags the code as a result.

I doubt this will resolve every alert, but I think it will help a lot and should shed some light on what specifically causes CodeQL to flag certain values which should make fixing any remaining alerts easier.

## How to test 🧪
We can lean on the automated test run to confirm that this does not break any tests. Aside from that, we can compare CodeQL results to see if and to what extent these changes help.
